### PR TITLE
feat: Add fiction warning

### LINKED ISSUE #259: Add Biocore is fictional to landing page
image

### PR DESCRIPTION
**Pull request by katjam**
Fixes #259 

## Description

- Add fiction banner to landing page

@TheLabCollective/developers

### CODE REVIEW

**Comment by katjam** [comment]
image

**Comment by katjam** [comment]
image

File: src/View/Landing.elm [hunk 1]
```diff
@@ -15,7 +15,13 @@ import View.Video
 view : Html Msg
 view =
     div [ class "landing container-fluid" ]
-        [ div [ class "container intro-video my-5" ]
+        [ div [ class "row justify-content-center" ]
+            [ div
+                [ class "col-6 my-4 alert alert-light text-center text-dark"
```
Comment (Nikomus) [inline src/View/Landing.elm:20]: Maybe let it be a little wider on a phone

```suggestion
                [ class "col-9 col-md-6 my-4 alert alert-light text-center text-dark"
```

**Review by Nikomus — APPROVED** [review]
Looks good! 🏑

**Pull request merged**